assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java

```diff
@@ -878,4 +878,7 @@ public String getDeepLinkWithPermissions(String context, String assignmentId, bo
      * @return A String containing the name of the content review service
      */
     public String getContentReviewServiceName();
+    
+    public String getUserAssignmentModify(String modifier);
```

Comment:
getUserAssignmentModify sounds weird to me, maybe 'getUserWhoModifiedAssignment' ?